app/src/docs/_examples/tables/ColumnFiltersTable.example.tsx

```diff
@@ -0,0 +1,153 @@
+        caption: 'Profile status',
+        info: 'This person profile status information',
+        render: (p) =>
+            p.profileStatus && (
+                <FlexRow>
+                    <Badge fill="transparent" color={ p.profileStatus.toLowerCase() as EpamAdditionalColor } caption={ p.profileStatus } />
+                </FlexRow>
+            ),
+        width: 140,
+        isSortable: true,
+        isFilterActive: (f) => !!f.profileStatusId,
+    }, {
+        key: 'salary',
+        caption: 'Salary',
+        render: (p) => (
+            <Text>
+                {getSeparatedValue(+p.salary, {
+                    style: 'currency', currency: 'USD', maximumFractionDigits: 2, minimumFractionDigits: 2,
+                }, 'en-US')}
+            </Text>
+        ),
+        width: 150,
+        textAlign: 'right',
+        isSortable: true,
+    }, {
+        key: 'jobTitle',
+        caption: 'Title',
+        render: (r) => <Text>{r.jobTitle}</Text>,
+        width: 200,
+        isFilterActive: (f) => !!f.jobTitleId,
+    }, {
+        key: 'birthDate',
+        caption: 'Birth date',
+        render: (p) => p?.birthDate && <Text>{dayjs(p.birthDate).format('MMM D, YYYY')}</Text>,
+        width: 120,
+        isSortable: true,
+    }, {
+        key: 'hireDate',
+        caption: 'Hire date',
+        render: (p) => p?.hireDate && <Text>{dayjs(p.hireDate).format('MMM D, YYYY')}</Text>,
+        width: 120,
+        isSortable: true,
+    },
+];
+
+export default function ColumnsConfigurationDataTableExample() {
+    const { api } = useUuiContext();
+
+    const filtersConfig = useMemo<TableFiltersConfig<Person>[]>(
+        () => [
+            {
+                field: 'profileStatusId',
+                columnKey: 'profileStatus',
+                title: 'Profile status',
+                type: 'multiPicker',
+                isAlwaysVisible: true,
+                dataSource: new LazyDataSource({ api: api.demo.statuses }),
+                predicates: defaultPredicates.multiPicker,
+                showSearch: false,
+            }, {
+                field: 'jobTitleId',
+                columnKey: 'jobTitle',
+                title: 'Title',
+                type: 'multiPicker',
+                dataSource: new LazyDataSource({ api: api.demo.jobTitles }),
+            }, {
+                field: 'salary',
+                columnKey: 'salary',
+                title: 'Salary',
+                type: 'numeric',
+                predicates: defaultPredicates.numeric,
+            }, {
+                field: 'birthDate',
+                columnKey: 'birthDate',
+                title: 'Birth date',
+                type: 'datePicker',
+            }, {
+                field: 'hireDate',
+                columnKey: 'hireDate',
+                title: 'Hire date',
+                type: 'rangeDatePicker',
+                predicates: defaultPredicates.rangeDatePicker,
+                presets: {
```

Comment:
let's remove presets customization from this example, to make it more simpler

Reply:
done